docs/usage/interpreter/interpreter_binding_mode.md

```diff
@@ -25,4 +25,97 @@ limitations under the License.
 
 ## Overview
 
+<center><img src="/assets/themes/zeppelin/img/docs-img/interpreter_binding_per_note_user.png" height="70%" width="70%"></center>
+<center><img src="/assets/themes/zeppelin/img/docs-img/interpreter_binding_scoped_isolated.png" height="70%" width="70%"></center>
 
+<br/>
+
+Interpreter is a JVM process that communicates to Zeppelin daemon using thrift. 
+Each Interpreter process can have Interpreter Groups, and each interpreter instance belongs to this Interpreter Group.
+(See [here](../../development/writing_zeppelin_interpreter.html) to understand more about its internal structure.) 
+
+Zeppelin provides 3 different modes to run interpreter process: **shared**, **scoped** and **isolated**.   
+Also, user can specify the scope of this mode as well: **per user** or **per note**.  
```

Comment:
1. Yeah. `Globally` means the `shared` mode. 
2. **The interpreter group will be instantiated** more makes sense.